### PR TITLE
API-31027 - Add LocalBGS to breakers

### PR DESCRIPTION
**Pull request by acovrig**
## Summary
- Enable breakers for BGS

## Related issue(s)
- [API-31027](https://jira.devops.va.gov/browse/API-31027)

## Testing done
- Did a GET of /v2/claims#index:
Got a 503 BadGateway the first time, the second request returned `Service unavailable`: `An outage has been reported on the BGS/Claims since timestamp UTC`. - is this the correct response(s)?

## What areas of the site does it impact?
Anything that uses LocalBGS

## Acceptance criteria
- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback
I'd like to have specs for this but that takes redis and IDK if we have that in the test pipeline.

### CODE REVIEW

File: config/initializers/breakers.rb [hunk 1]
```diff
@@ -80,7 +80,8 @@
     IAMSSOeOAuth::Configuration.instance.breakers_service,
     CovidVaccine::V0::VetextConfiguration.instance.breakers_service,
     VEText::Configuration.instance.breakers_service,
-    PagerDuty::Configuration.instance.breakers_service
+    PagerDuty::Configuration.instance.breakers_service,
+    ClaimsApi::LocalBGS.breakers_service
```
Comment (FonzMP) [inline config/initializers/breakers.rb:84]: I see all of these have `.instance` but ours does not? Did we miss this or is it not needed?

Comment (acovrig) [reply config/initializers/breakers.rb:84]: it depends, the `EVSS::ClaimsService` above doesn't - I did a `self.breakers_service` in the `LocalBGS` so I don't need `.instance` - I don't think there's any benefit either way for it? The `LocalBGS` initializer takes 2 args, I wonder if those would still be required if I did `.instance`.

**Review by ericboehs — APPROVED** [review]

**Comment by rmtolmach** [comment]
@acovrig To get the **Validate changes to DataDog Service Catalog Files** check passing, please update your branch with the latest changes from `master`. Thanks!

**Review by FonzMP — APPROVED** [review]
:shipit:

**Pull request merged**